sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Blob/ArtifactManifest.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Containers.ContainerRegistry.Specialized
+{
+    /// <summary>
+    /// </summary>
+    public abstract class ArtifactManifest
```

Comment:
Description for this class is missing.

Reply:
@ShivangiReja, `ArtifactManifest` now gets its doc comment from the generator, see [ArtifactManifest in APIView]( https://apiview.dev/Assemblies/Review/38848951c47540cbbd20dc2c969cca9e/25d11a3d978b4520bdd04a9860a5e6ad?doc=True&diffOnly=False#Azure.Containers.ContainerRegistry.Specialized.ArtifactManifest)

We can improve these doc comments in the swagger or in swagger transforms as needed.